keyhive_core/src/principal/individual/state.rs

```diff
@@ -304,121 +246,4 @@ mod tests {
         assert!(built.contains(&share_key_3));
         assert!(built.contains(&share_key_5));
     }
-
-    #[test]
-    fn test_causal_delivery() {
```

Comment:
This is yet another of those things from the demo branch that we need to bring across. We don't depend on causal delivery for prekeys because we don't want to block new keys.